downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,422 @@
+| `getCompletionType`                                                       | `io.ballerina.runtime.api.types.StreamType`        |
+| `getConstrainedType`                                                      | `io.ballerina.runtime.api.types.TableType`         |
+| `getTupleTypes`                                                           | `io.ballerina.runtime.api.types.TupleType`         |
+| `getRestType`                                                             | `io.ballerina.runtime.api.types.TupleType`         |
+| `getConstraint`                                                           | `io.ballerina.runtime.api.types.TypedescType`      |
+| `getMemberTypes`                                                          | `io.ballerina.runtime.api.types.UnionType`         |
+| `getOriginalMemberTypes`                                                  | `io.ballerina.runtime.api.types.UnionType`         |
+| `getElementType`                                                          | `io.ballerina.runtime.api.values.BArray`           |
+| `getConstraintType`                                                       | `io.ballerina.runtime.api.values.BStream`          |
+| `getCompletionType`                                                       | `io.ballerina.runtime.api.values.BStream`          |
+| `getKeyType`                                                              | `io.ballerina.runtime.api.values.BTable`           |
+| `getDescribingType`                                                       | `io.ballerina.runtime.api.values.BTypedesc`        |
+| `getType`                                                                 | `io.ballerina.runtime.api.values.BValue`           |
+| `getType`                                                                 | `io.ballerina.runtime.api.utils.TypeUtils`         |
+
+For example, if an intersection type is defined in the following way,
+
+```ballerina
+type ReadonlyIntArray readonly & int[];
+
+ReadonlyIntArray array = [1, 2, 3, 4];
+```
+
+the results of the runtime API calls will be as follows.
+
+| **Runtime API call**                                   | **Result**                                                     |
+|--------------------------------------------------------|----------------------------------------------------------------|
+| `array.getType()`                                      | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getDescribingType()` on `ReadonlyIntArray` type       | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getImpliedType()` on received `IntersectionType` type | This will return an `ArrayType`.                               |      
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
```

Comment:
We no longer say Standard library. It should be Ballerina library.

Reply:
Fixed.